libraries/AP_HAL_Linux/HAL_Linux_Class.cpp

```diff
@@ -320,11 +320,13 @@ void HAL_Linux::run(int argc, char* const argv[], Callbacks* callbacks) const
         {"storage-directory",   true,  0, 's'},
         {"module-directory",    true,  0, 'M'},
         {"defaults",            true,  0, 'd'},
+        {"affinity-num",        true, 0, 'a'},
+        {"affinity-start",      true, 0, 'f'},
```

Comment:
see suggestion below. Maybe just one --cpu-affinity param?

Reply:
done .. and removed